test/unit/test.database.js

```diff
@@ -1,13 +1,247 @@
+    miner: "0x0000000000000000000000000000000000000000",
+    mixHash: "0x0000000000000000000000000000000000000000000000000000000000000000",
+    nonce: "0x0000000000000000",
+    number: hexify(number || 2),
+    parentHash: "0xb23952935adf39ec4198575fb16cdffe54be970fba0e264eb900896dcae7e2b0",
+    receiptsRoot: "0x54f1a38f3fde228e85dab1f53f31be6eeb06391cc98ff7110819adf0907bfd92",
+    sha3Uncles: "0x1dcc4de8dec75d7aab85b567b6ccd41ad312451b948a7413f0a142fd40d49347",
+    size: "0x12ce",
+    stateRoot: "0x840d89f25125ecbc0c308be4ec99376be568fead2a791907b3493ea0810e4e21",
+    timestamp: hexify(10000000),
+    totalDifficulty: "0x4d903e",
+    transactions: [
+      "0xef88f2cef652f5005348ff99c68c0152e2dda275753fe1cfc50f912f24d8aabf",
+      "0x5e7013fdcdcf6da910a30c69ca2545e1ab840e724f0d45c13be44bc1d13bca38",
+      "0xec40460e77d3e8d1524e2f0cca34cd1235f6cf2d0f8e2669bf7dd1454f59a89a",
+      "0xf6a187a2f690e214c81d549ca7fb581e36633de81746eeb72d66d2ac961e924a",
+      "0x3c6c8b4dd0f9738f1eca9f398f834007fc112b7134c861d5763f6143c4b06bc2",
+      "0xe09c5e91c11ca80626c4bb2767469f3b5bfd62fa061e0cf21af55b977d638fba",
+      "0x653a76c8e360484840093e0c53f397bf2f2cb1e4a0e9b1e82c3207a91255377a",
+      "0x3c582338674000169bef47f43dd65190418e395a4b9e13108a5c74f99bdfff49",
+      "0x5a3ba6d658a61d06d2efc2e6a4cdc17e89b789d03d92c9ac2be78cdf16118c6f",
+      "0x5b27bf1f9bdd7afe7b76a59305b1674b2d65294d7bb075a259dfc838357f26d9",
+    ],
+    transactionsRoot: "0x2ad5b79ead3919e9a7ab6cdc9aa4034ef9d1b147ea5deefec90c2b8a9e2f5eaa",
+    uncles: [],
+  }, overrides || {});
+}
+
+function makeMockAugur(additional) {
+  const augur = new Augur();
+  augur.api = {
+    Universe: {
+      getReputationToken: () => "REP_TOKEN",
+      getOrCacheReportingFeeDivisor: () => 1,
+    },
+    Market: {
+      getFeeWindow: () => "FEE_WINDOW",
+      getEndTime: () => 592903560,
+      getDesignatedReporter: () => "DESIGNATED_REPORTER",
+      getMarketCreatorMailbox: () => "MARKET_CREATOR_MAILBOX",
+      getNumTicks: () => 42,
+      getMarketCreatorSettlementFeeDivisor: () => 0.01,
+      getValidityBondAttoeth: () => "47",  // must be a number
+      getShareToken: args => Promise.resolve(`SHARE_TOKEN_${args._outcome}`),
+    },
+  };
+  augur.rpc = {
+    getNetworkID: () => "9090",
+  };
+  augur.contracts = {
+    addresses: {
+      9090: {
+      },
+    },
+  };
+
+  return deepmerge(augur, additional || {});
+}
+
+const setupTestDb = (augur, logs, blockDetails) => {
```

Comment:
Since all the other functions in this file are defined as standard functions, consider making this one consistent with the others?

Reply:
Oh yeah good catch. I added the other functions but only modified this one so it slipped past me.